kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -1,38 +1,12 @@
-define(['bluebird', 'kb_service/client/workspace', 'kb_service/utils'], (
+define(['bluebird', 'kb_service/client/workspace', 'kb_service/utils', 'common/lang'], (
```

Comment:
it's hard to tell from looking at the PR files, but how do the contents of this file differ from `kbase-extension/static/kbase/js/common/validation.js`? A number of the function names are the same.

Reply:
Sounds good.